application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -1036,7 +1036,7 @@ type Place {
   Type of vertex. (Normal, Bike sharing station, Bike P+R, Transit stop) Mostly
   used for better localization of bike sharing and P+R station names
   """
-  vertexType: VertexType
+  vertexType: VertexType @deprecated(reason : "Unmaintained. Use `stop`, `vehicleParking` or `vehicleRentalStation` to tell which type it is.")
```

Comment:
Isn't `rentalVehicle` also one alternative?

Reply:
Yes, you're correct.